exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -273,10 +273,12 @@ where
     // Retrieve config by key
     let config = service_config
         .get(key)
-        .expect(&format!(
-            "Unable to read config with key [{}] from node configuration",
-            key
-        ))
+        .unwrap_or_else(|| {
+            panic!(
```

Comment:
Is it to format things lazily?

Reply:
Yes, this prevents [unconditional call of functions inside .expect()](https://rust-lang.github.io/rust-clippy/master/index.html#expect_fun_call). First, I ended up with `#[allow(clippy::expect_fun_call)]`, but then I decided that it's better to fix a couple of lines.

Reply:
good, thanks!